plugins/modules/vmware_guest.py

```diff
@@ -923,6 +923,48 @@
         touch /tmp/touch-from-playbook
   delegate_to: localhost
 
+# you can initialize the metadata template to a varable as
+# follows.
```

Comment:
why is this task commented additionally? this is already a comment block.

Reply:
your example task references `vm_cloud_init_metadata` and `vm_cloud_init_userdata`.

Your commented example sets `vm_cloud_init_metadata`, but doesn't set `vm_cloud_init_userdata`.